python/ray/tune/search/vizier/vizier_search.py

```diff
@@ -0,0 +1,248 @@
+"""A Vizier Ray Searcher."""
+
+import datetime
+import json
+from typing import Dict, Optional
+import uuid
+
+from ray.tune.result import DEFAULT_METRIC
+from ray.tune.search.variant_generator import parse_spec_vars
+from ray.tune.search import (
+    UNDEFINED_METRIC_MODE,
+    UNRESOLVED_SEARCH_SPACE,
+    Searcher,
+)
+
+# Make sure that importing this file doesn't crash Ray, even if Vizier wasn't installed.
+try:
+    from vizier import raytune as vzr
+    from vizier.service import clients
+    from vizier.service import pyvizier as svz
```

Comment:
nit: Are these abbreviations are commonly used in Vizier? If not maybe we can use more explicit naming conventions like `vizier_raytune` and just `pyvizier`.

Reply:
These are pretty common in Vizier (i.e. `pyvizier` is either `vz` or `svz` depending if it's the common API or the service-specific API). The abbreviation greatly shortens primitive constructions typically - I'd recommend keeping them here unless Ray really prefers not to.